packages/create-package/template/src/MyFirstComponent.ts

```diff
@@ -40,11 +44,11 @@ class INIT_PACKAGE_VAR_CLASS_NAME extends UI5Element {
 	}
 
 	static async onDefine() {
-		INIT_PACKAGE_VAR_CLASS_NAME.i18nBundle = await getI18nBundle("INIT_PACKAGE_VAR_NAME");
```

Comment:
Why is this changed to test?

Reply:
It was copied by mistake from one of the new created packages for test cases. its restored.